generate_ghidra_pyi.py

```diff
@@ -50,6 +50,45 @@ def main():
     ghidra_package = type_extractor.Package.from_package(ghidra)
     type_formatter.create_type_hints(pyi_root, ghidra_package)
 
+    setup_code = """
+from setuptools import setup
+import os
+
+packages = ['ghidra-stubs']
+
+package_data = {'': ['*']}
+
+
+
+def find_stub_files():
+    result = []
+    for root, dirs, files in os.walk('ghidra-stubs'):
+        for file in files:
+            if file.endswith('.pyi'):
+                if os.path.sep in root:
+                    sub_root = root.split(os.path.sep, 1)[-1]
+                    file = os.path.join(sub_root, file)
+                result.append(file)
+    return result
+
+setup_kwargs = {
+'name': 'ghidra-stubs',
+    'version': '%s',
+    'author': 'Tamir Bahar',
+    'packages': packages,
+    'package_data': {'ghidra-stubs': find_stub_files()},
+}
+
+
+setup(**setup_kwargs)
+""" % (getGhidraVersion())
+
+    import os
+    os.rename(os.path.join(pyi_root, 'ghidra'), os.path.join(pyi_root, 'ghidra-stubs'))
```

Comment:
Why is the rename needed?
Can't we just change the directory name in `setup.py`?

Reply:
I do not know packaging for Python that well in that regard. I can rename every reference beside the `name` parameter in setup to `ghidra`. Looking at `site-packages` this leads to a folder `ghidra` in there. Pycharm at least still recognizes this. There is probably never going to be a python package that provides a `ghidra` module, but if there is they would end up in the same folder and theoretically conflict with package provided type hints. I think this is why you aren't supposed to do that according to https://www.python.org/dev/peps/pep-0561/#stub-only-packages. The [example](https://github.com/ethanhs/stub-package/blob/master/setup.py) has everything named `-stub`.  Is there some way to alias that folder in the setup.py? That would definitely be cleaner IMO.